doc/troubleshooting.md

```diff
@@ -29,6 +29,12 @@ Then simply restart the job server.
 
 Note that the idle-timeout must be higher than request-timeout, or Spray and the job server won't start.
 
+## Timeout getting large job results
+
+If your job returns a large job result, it may exceed Akka's maximum network message frame size, in which case the result is dropped and you may get a network timeout.  Change the following configuration, which defaults to 10 MB:
+
+    akka.remote.netty.tcp.maximum-frame-size = 100 MiB
```

Comment:
MB = 1024 * 1024;  MiB = 1000000

> On Feb 25, 2016, at 1:03 AM, Noorul Islam K M <notifications@github.com> wrote:
> 
> In doc/troubleshooting.md <https://github.com/spark-jobserver/spark-jobserver/pull/390#discussion_r54065399>:
> 
> > @@ -29,6 +29,12 @@ Then simply restart the job server.
> >  
> >  Note that the idle-timeout must be higher than request-timeout, or Spray and the job server won't start.
> >  
> > +## Timeout getting large job results
> > +
> > +If your job returns a large job result, it may exceed Akka's maximum network message frame size, in which case the result is dropped and you may get a network timeout.  Change the following configuration, which defaults to 10 MB:
> > +
> > +    akka.remote.netty.tcp.maximum-frame-size = 100 MiB
> Is MiB == MB?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/spark-jobserver/spark-jobserver/pull/390/files#r54065399>.
>